Pod/Classes/ios/NYTPhotoViewController.m

```diff
@@ -157,8 +157,8 @@ - (void)didDoubleTapWithGestureRecognizer:(UITapGestureRecognizer *)recognizer {
     CGPoint pointInView = [recognizer locationInView:self.scalingImageView.imageView];
     
     CGFloat newZoomScale = self.scalingImageView.maximumZoomScale;
-    
-    if (self.scalingImageView.zoomScale >= self.scalingImageView.maximumZoomScale) {
+
+    if ((self.scalingImageView.zoomScale - self.scalingImageView.maximumZoomScale) >= 0.0) {
```

Comment:
@cdzombak  I don't think this works. I've been in a situation where `self.scalingImageView.zoomScale` is slightly smaller than `self.scalingImageView.maximumZoomScale`(well, not really small but strict-float-comparison small). That's why I add the `ABS` to deal with it.